nostotagging.php

```diff
@@ -188,64 +204,83 @@ public function uninstall()
 	 */
 	public function getContent()
 	{
+		// We don't want the browser to cache the config page, as the iframe authentication won't work if cached.
+		header('Cache-Control: no-store, no-cache, must-revalidate, max-age=0');
+		header('Cache-Control: post-check=0, pre-check=0', false);
+		header('Pragma: no-cache');
```

Comment:
Actually, the need to be in the module because the browser will cache the iframe element, including it's src attribute which includes one time authentication tokens, that change every request.